kfs-core/src/main/resources/edu/arizona/kfs/fp/ojb-fp.xml

```diff
@@ -1,243 +1,352 @@
 <descriptor-repository version="1.0">
 
-  <!-- DV needs a new class to accept,  sourceAccountingLines use new Accounting Line class -->

```

Comment:
Looks like the IDE's formatter got a hold of this file, i.e., some places with mixed 2 and 4 spaced indentation has been replaced by 4 spaces. This makes it difficult to diff depending on the tool being used. Generally we would want to leave formatting alone so we can more easily see the meaningful changes. (An exception to this is initial commit, which is when we'd want to apply formatting to the entire file).